locals.tf

```diff
@@ -696,6 +696,15 @@ locals {
       }
     }
 
+    eventbridge = {
```

Comment:
Looks good, but please add a comment right above your addition, like this:
```
# https://docs.aws.amazon.com/step-functions/latest/dg/eventbridge-iam.html
```

It helps to validate and to understand the relations.